.gitignore

```diff
@@ -19,7 +19,7 @@
 /.metadata/*
 *.eml
 /run
-/runServer
+/run_server

```

Comment:
I'm happy for `run_server` to be added, but I would prefer if the old one was there as well